### ISSUE TITLE
Be great to get this dockerized and up on docker hub

### ISSUE THREAD

**Issue by freebo** [tvshowtime/tvshowtime-plex-scrobbler#19]

**Comment by surdu** [comment]
I could give this a try. I've read on Plex forum that this is not working anymore though. Is that right ?

**Comment by ghost** [comment]
Yeah, from what I can tell. I know it only supports o e account though, so
no matter who watches it, it marks it off on the same account. Would be
nice to be able to have it work per-user instead of per-server.
